modules/schematics/src/effect/index.spec.ts

```diff
@@ -50,6 +50,44 @@ describe('Effect Schematic', () => {
     ).toBeGreaterThanOrEqual(0);
   });
 
+  it('should create an effect class which has an "effect$" property if feature is set', () => {
+    const options = { ...defaultOptions, feature: true };
+
+    const tree = schematicRunner.runSchematic('effect', options, appTree);
+    const content = tree.readContent(
+      `${projectPath}/src/app/foo/foo.effects.ts`
+    );
+    expect(content).toMatch(
+      /import { Actions, Effect, ofType } from '@ngrx\/effects';/
+    );
+    expect(content).toMatch(
+      /import { LoadFoos, FooActionTypes } from '\.\/foo.actions';/
+    );
+    expect(content).toMatch(/export class FooEffects/);
+    expect(content).toMatch(
+      /effect\$ = this\.actions\$.pipe\(ofType<LoadFoos>\(FooActionTypes\.LoadFoos\)\);/
```

Comment:
Maybe change `effects$` to be `loadFoos$`?

Reply:
Thank you for your comment!
I renamed "effect$" to "loadFoos$".
It's more suitable in this case.